bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,66 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+
+    See <https://github.com/btford/write-good> for more information.
+    """
+    LANGUAGES = {"Natural Language"}
+    REQUIREMENTS = {NpmRequirement('write-good', '0.9.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         passive: bool=False,
+                         illusion: bool=False,
+                         adverb: bool=False,
+                         so: bool=False,
+                         thereIs: bool=False,
+                         weasel: bool=False,
+                         tooWordy: bool=False,
```

Comment:
These are command line arguments, I have never seen snake_case as CLI arguments. BTW, this is the naming which the official docs followed, so I thought it would be sane to use the same thing as the users will be familiar with it. Your thoughts ?